src/main/java/com/solace/tools/solconfig/Commander.java

```diff
@@ -32,8 +32,9 @@ public ConfigBroker backup(String resourceType, String[] objectNames, boolean is
         log.info("Starting broker config backup: resourceType={}, objectNames={}, isKeepDefault={}",
                 resourceType, objectNames, isKeepDefault);
         exitOnObjectsNotExist(resourceType, objectNames);
+        log.info("Generating broker config");
```

Comment:
Nit - Since this comment is for the purpose of debugging, shall we consider lower the logging level to `debug`?

Reply:
This PR might end up changing a bit.

For some reason debug logs don't show and it's stopping us from debugging in production. So, we're upgrading these to info.